packages/psss/src/api/queries/helpers/query.js

```diff
@@ -44,11 +44,12 @@ export const useReport = (endpoint, apiOptions = {}, queryOptions = {}) => {
  * @param queryOptions react-query options
  */
 export const usePaginatedReport = (endpoint, apiOptions = {}, queryOptions = {}) => {
-  const query = usePaginatedQuery([endpoint, apiOptions.params], () => get(endpoint, apiOptions), {
+  const query = useQuery([endpoint, apiOptions.params], () => get(endpoint, apiOptions), {
```

Comment:
`usePaginatedQuery` has been replaced by `keepPreviousData: true`, see: https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-3#usepaginatedquery-has-been-removed-in-favor-of-the-keeppreviousdata-option